manual/src/full---help-output.md

```diff
@@ -9,7 +9,8 @@ Arguments:
   [MINUS_FILE]
           First file to be compared when delta is being used in diff mode
 
-          `delta file_1 file_2` is equivalent to `diff -u file_1 file_2 | delta`.
+          `delta file_1 file_2` is equivalent to `diff -u file_1 file_2 |
+          delta`.
 
```

Comment:
Now that is annoying, doesn't fit in 80 lines!

Reply:
I'd be happy to lose the underscores; I think I'd debated them once or twice with myself.

Reply:
Nice, now it fits!